kolibri/plugins/document_epub_render/assets/src/views/SettingsSideBar.vue

```diff
@@ -174,11 +180,15 @@
       textSize: 'Text size',
       decrease: 'Decrease',
       increase: 'Increase',
+      decreaseTextSize: 'Decrease text size',
+      increaseTextSize: 'Increase text size',
       theme: 'Theme',
       setWhiteTheme: 'Set white theme',
       setBeigeTheme: 'Set beige theme',
       setGreyTheme: 'Set grey theme',
       setBlackTheme: 'Set black theme',
+      setHighContrastBlackTheme: 'Set high contrast black theme',
+      setHighContrastWhiteTheme: 'Set high contrast white theme',
```

Comment:
We're past string freeze for the `release-v0.12.y` branch, so we can't add or modify strings in this PR. We can make a small follow-up PR targeting `develop`

Reply:
Oh okay I remember that being announced but I thought this was going into a different release. I'm a bit confused haha.

Reply:
* 0.12.3 -> upcoming release, no string changes
* 0.12.4 / y -> upcoming "bigger" release with full integration test pass and string changes, currently being translated
* 0.13.0 -> next big feature release

Reply:
So "currently being translated" means no string additions?

Reply:
correct - usually we call that "string freeze" but I think that needs to be renamed :)